kermit/build.gradle.kts

```diff
@@ -57,11 +57,16 @@ kotlin {
     val commonMain by sourceSets.getting
     val commonTest by sourceSets.getting
 
+    val commonJvmMain by sourceSets.creating
+    commonJvmMain.dependsOn(commonMain)
```

Comment:
It's probably also worth configuring a `commonJvmTest` even if there's nothing in it at the moment, just so it doesn't get confusing later if we try to add test sources there.

Reply:
done